tools/gz.cc

```diff
@@ -482,7 +483,26 @@ bool ModelCommand::RunImpl()
       return false;
     }
 
-    return this->ProcessSpawn(sdf, modelName, pose, node);
+    std::string resolvedFileName = filename;
+    if (common::isDirectory(filename))
+    {
+      // Use sdf::getModelFilePath() instead of filename because filename
+      // might be a model rectory
+      resolvedFileName = sdf::getModelFilePath(filename);
+      if (resolvedFileName.empty())
+      {
+        std::cerr
+            << "The provided file is a directory, but a model could not found"
```

Comment:
> could not found

could not *be* found

Reply:
https://github.com/osrf/gazebo/pull/2734/commits/60d082bcdde5f998286b46c7957ca14a2988b1cb